cuda/runtime/BUILD.local_cuda

```diff
@@ -11,7 +11,9 @@ filegroup(
         ":_cuda_header_files",
     ] + glob([
         "cuda/version.txt",
-        "cuda/bin/**",
+        "cuda/bin/nvcc",
+        "cuda/bin/nvlink",
+        "cuda/bin/bin2c",
```

Comment:
I suspect we also need ptxas here?

Reply:
I just swapped out the design to bring back the **, which should fix this issue and make the code a bit less fragile.